Makefile

```diff
@@ -12,6 +12,6 @@ vendordocs:
 	VENDOR_API="${SOURCE}" ./vendor.sh
 
 setup:
-	mkdir java
+	mkdir java || true
```

Comment:
Ha, I honestly don't think I ever knew that option existed! I will update the line, thanks.